evm/contracts/HTLC/HTLCBase.sol

```diff
@@ -108,7 +108,7 @@ abstract contract HTLCBase is IHTLC {
             revert TooLate();
```

Comment:
Since there is now a signature allowing EVM contract to withdraw or refund when in a signed HTLC. The date control for withdraw in signedHTLC on EVM should be removed, otherwise we could have this case:
- User create signed HTLC on Archethic and EVM.
- User withdraw on Archethic and get withdraw secret and signature for EVM
- User wait for any reason and pass the lockTime
- User cannot withdraw EVM (TooLate error), User cannot refund on EVM (InvalidSignature error)

Reply:
Yes I can remove it for signed HTLC, but still keep for chargeable as EVM is the originated source